src/Equations/damaged-elastic/Kernels/Time.cpp

```diff
@@ -225,19 +223,18 @@ void seissol::kernels::Time::computeAder(double i_timeStepWidth,
   }
 
   for (unsigned int q = 0; q < NUMBER_OF_ALIGNED_BASIS_FUNCTIONS; ++q) {
-    real EspI = (exxNodal[q] + epsInitxx) + (eyyNodal[q] + epsInityy) + (ezzNodal[q] + epsInitzz);
-    real EspII = (exxNodal[q] + epsInitxx) * (exxNodal[q] + epsInitxx) +
-                 (eyyNodal[q] + epsInityy) * (eyyNodal[q] + epsInityy) +
-                 (ezzNodal[q] + epsInitzz) * (ezzNodal[q] + epsInitzz) +
-                 2 * (exyNodal[q] + epsInitxy) * (exyNodal[q] + epsInitxy) +
-                 2 * (eyzNodal[q] + epsInityz) * (eyzNodal[q] + epsInityz) +
-                 2 * (ezxNodal[q] + epsInitzx) * (ezxNodal[q] + epsInitzx);
-    real xi;
-    if (EspII > 1e-30) {
-      xi = EspI / std::sqrt(EspII);
-    } else {
-      xi = 0.0;
-    }
+    real EspI, EspII, xi;
+    calculateEps(exxNodal,
+                 eyyNodal,
+                 ezzNodal,
+                 exyNodal,
+                 eyzNodal,
+                 ezxNodal,
+                 q,
+                 *m_damagedElasticParameters,
```

Comment:
Make this argument a pointer, then you don't have to dereference in order to pass by reference.

Reply:
Done